code/modules/multiz/pipes.dm

```diff
@@ -19,7 +19,11 @@
 	var/maximum_pressure = 70*ONE_ATMOSPHERE
 	var/fatigue_pressure = 55*ONE_ATMOSPHERE
 	alert_pressure = 55*ONE_ATMOSPHERE
-
+	
+	var/travel_verbname = "UNDEFINED"
```

Comment:
I think these can be `var/static/`. @Lohikar will this work properly with childed classes or not?